models/gc_spider_baseline/meta.json

```diff
@@ -0,0 +1,219 @@
+    "zenodo": "",
+    "colab": "",
+    "slicer": false
+  },
+  "info": {
+    "use": {
+      "title": "Intended use",
+      "text": "This model is intended to be used with T1 or T2 weighted MRI images of the lumbar spine in sagittal orientation only. Ensure that the lumbar spine is visible in the input MRI images to ensure reliable results. This algorithm is available on Grand Challenge [1] and is used as baseline for the SPIDER challenge [2].",
+      "references": [
+        {
+          "label": "SPIDER baseline algorithm on grand-challenge",
+          "uri": "https://grand-challenge.org/algorithms/spider-baseline-iis/"
+        },
+        {
+          "label": "SPIDER challenge details",
+          "uri": "https://spider.grand-challenge.org/"
+        }
+      ],
+      "tables": []
+    },
+    "analyses": {
+      "title": "Evaluation",
+      "text": "The segmentation performance was evaluated as part of the SPIDER challenge using two metrics: (1) The Dice coefficient to measure the volume overlap, and (2) the average absolute surface distance (ASD) as an indication of the segmentation accuracy along the surface of all structures.",
+      "references": [
+        {
+          "label": "SPIDER baseline algorithm evaluation results on grand-challenge",
+          "uri": "https://spider.grand-challenge.org/evaluation/11bf4d89-6768-4e2f-b540-60178ba33a95/"
+        }
+      ],
+      "tables": [
+        {
+          "label": "Dice score.",
+          "entries":
+          {
+            "Vertebrae": "0.93 ± 0.05",
+            "Intervertebral discs": "0.84 ± 0.10",
+            "Spinal canal": "0.92 ± 0.04"
+          }
+        },
+        {
+          "label": "ASD (mm).",
+          "entries":
+          {
+            "Vertebrae": "0.48 ± 0.95",
+            "Intervertebral discs": "0.54 ± 0.45",
+            "Spinal canal": "0.39 ± 0.45"
+          }
+        }
+      ]
+    },
+    "evaluation": {
+      "title": "Evaluation data",
+      "text": "The test set consists of 87 lumbar MRI images from 39 patients, which includes the fully manually annotated images of 15 patients, randomly selected from the set of fully annotated dataset, see Training data.",
+      "references": [],
+      "tables": []
+    },
+    "training": {
+      "title": "Training data",
+      "text": "For training a large, publicly available, multi-center, lumbar spine MRI dataset was used. It includes reference segmentations of vertebrae, intervertebral discs (IVDs), and spinal canal. The dataset includes 447 sagittal T1 and T2 MRI images from 218 patients with a history of low back pain. For 20 patients from the UMC dataset the images were fully, manually annotated. This model is based on a previously developed model trained on both CT and MRI data [1]. It was extended by also segmenting the IVDs and spinal canal, using MRI only. For training details see Baseline 1 in the associated paper [2].",
```

Comment:
We should also add the link to the dataset as it's mentioned to be public: 
https://doi.org/10.5281/zenodo.10159290

Reply:
Done